src/auto/about_dialog.rs

```diff
@@ -383,10 +383,11 @@ impl AboutDialogBuilder {
         if let Some(ref width_request) = self.width_request {
             properties.push(("width-request", width_request));
         }
-        glib::Object::new(AboutDialog::static_type(), &properties)
+        let ret = glib::Object::new(AboutDialog::static_type(), &properties)
             .expect("object new")
-            .downcast()
-            .expect("downcast")
+            .downcast::<AboutDialog>()
+            .expect("downcast");
+        ret
```

Comment:
Poor clippy :)

Reply:
Yeah clippy will get sad, but also not, because we already override that one I think.

Reply:
I'll rebase onto current master to make that clearer though.